src/commands/get-woocommerce-stats.php

```diff
@@ -0,0 +1,219 @@
+			$progress_bar = new ProgressBar( $output, $site_count );
+			$progress_bar->start();
+			// Checking each site for the plugin slug: woocommerce, and only saving the sites that have it active
+			foreach ( $site_list as $site ) {
+				$progress_bar->advance();
+				$plugin_list = $this->get_list_of_plugins( $site[0] );
+				if ( ! is_null( $plugin_list ) ) {
+					if ( ! is_null( $plugin_list->data ) ) {
+						$plugins_array = json_decode( json_encode( $plugin_list->data ), true );
+						foreach ( $plugins_array as $plugin_path => $plugin ) {
+							$folder_name = strstr( $plugin_path, '/', true );
+							$file_name   = str_replace( array( '/', '.php' ), '', strrchr( $plugin_path, '/' ) );
+							if ( ( $plugin_slug === $plugin['TextDomain'] || $plugin_slug === $folder_name || $plugin_slug === $file_name ) && $plugin['active'] == 'Active' ) {
+								$sites_with_woocommerce[] = array( $site[1], $site[0] );
+							}
+						}
+					}
+				}
+			}
+			$progress_bar->finish();
+			$output->writeln( '<info>  Yay!</info>' );
+
+		} else {
+			// Get plugin lists from Jetpack profile data
+			$output->writeln( '<info>Checking Jetpack site profiles for WooCommerce...<info>' );
+			$jetpack_sites_plugins = $api_helper->call_wpcom_api( 'rest/v1.1/me/sites/plugins/', array() );
+
+			foreach ( $site_list as $site ) {
+				//check if the site exists in the jetpack_sites_plugins object
+				if ( isset( $jetpack_sites_plugins->sites->{$site[0]} ) ) {
+					// loop through the plugins and check for WooCommerce
+					foreach ( $jetpack_sites_plugins->sites->{$site[0]} as $site_plugin ) {
+						if ( $site_plugin->slug === $plugin_slug && $site_plugin->active === true ) {
+							$sites_with_woocommerce[] = array( $site[1], $site[0] );
+						}
+					}
+				}
+			}
+		}
+
+		$woocommerce_count = count( $sites_with_woocommerce );
+		$output->writeln( "<info>{$woocommerce_count} sites have WooCommerce installed and active.<info>" );
+
+		// Get WooCommerce stats for each site
+		$output->writeln( '<info>Fetching WooCommerce stats for Team51 production sites...<info>' );
+		$progress_bar = new ProgressBar( $output, $woocommerce_count );
+		$progress_bar->start();
+
+		$team51_woocommerce_stats = array();
+		foreach ( $sites_with_woocommerce as $site ) {
+			$progress_bar->advance();
+			$stats = $this->get_woocommerce_stats( $site[1], $unit, $date );
+
+			//Checking if stats are not zero. If not, add to array
+			if ( isset( $stats->total_gross_sales ) && $stats->total_gross_sales > 0 && $stats->total_orders > 0 ) {
+				array_push( $team51_woocommerce_stats, array( $site[0], $site[1], $stats->total_gross_sales, $stats->total_net_sales, $stats->total_orders, $stats->total_products ) );
+			}
+		}
+		$progress_bar->finish();
+		$output->writeln( '<info>  Yay!</info>' );
+
+		//Sort the array by total gross sales
+		usort(
+			$team51_woocommerce_stats,
+			function ( $a, $b ) {
+				return $b[2] - $a[2];
```

Comment:
Consider building this as an associative array so the comparison being done is clear.